components/dev-tools/spack/SPECS/spack.spec

```diff
@@ -41,6 +41,7 @@ Requires: python2-mock
 DocDir:    %{OHPC_PUB}/doc/contrib
 
 %global install_path %{OHPC_ADMIN}/%{pname}/%version
+%define spack_install_path %{OHPC_PUB}/%{pname}/%version
```

Comment:
Please use %global and not %define, in this case %define works, but we try to go with %global everywhere

Reply:
Done - thank for the heads up! Didn't know define could lead to weird behaviour.

Reply:
If it really leads to weird behaviour is heavily discussed. But %define has a different scope than %global and to avoid ambiguities, sticking to %global is easier.